### PR TITLE
Cluster Restart and Stop

### PR DESCRIPTION
**Pull request by fsat**
- Move Cluster Restart into its own page from Cluster Troubleshooting page.
- Introduce Cluster Stop page since the stop instruction is slightly different from the Cluster Restart.

### CODE REVIEW

**Comment by fsat** [comment]
Needs to be cherry-picked to 2.0 as well.

**Review by markusjura — APPROVED** [review]

**Comment by edwardcallahan** [comment]
Particularly for DC/OS, should we note in the installation guide to see 'Stop Cluster' for uninstall instructions?

File: src/main/play-doc/operation/Install.md [hunk 1]
```diff
@@ -812,6 +812,8 @@ Wait until the ConductR instance's health is marked as healthy before proceeding
 
 If more instances are required, scale ConductR to the desired total schedulers using the `Instances` within `Edit Service` not the `Scale` dialog. Ensure that all of the instances are healthy before proceeding. Multiple core schedulers are recommended for resilience however schedulers are not required on all nodes.
 
+_To stop a deployed and running ConductR in DC/OS cluster, be sure to follow the instructions on the [Cluster Stop](ClusterStop#DC/OS-mode) page._
```
Comment (fsat) [inline src/main/play-doc/operation/Install.md:815]: @edwardcallahan - link added

**Comment by markusjura** [comment]
> Particularly for DC/OS, should we note in the installation guide to see 'Stop Cluster' for uninstall instructions?

@edc Yes, I like that. I am currently updating our doc in regards to DC/OS and will do this change within this PR as well.

**Comment by fsat** [comment]
#340 takes care of `2.0`

**Pull request merged**